boefjes/boefjes/plugins/kat_dns/schema.json

```diff
@@ -10,10 +10,11 @@
     },
     "REMOTE_NS": {
       "title": "REMOTE_NS",
-      "maxLength": 45,
+      "maxLength": 15,
```

Comment:
I chose 45 to also allow for ipv6 nameserver ip's.

Reply:
Addressed in 7020509b021935dcf87f20cc0fab976f4f69a37e.